rocAL/rocAL/source/rali_api_meta_data.cpp

```diff
@@ -77,13 +77,21 @@ RALI_API_CALL raliCreateVideoLabelReader(RaliContext p_context, const char* sour
 }
 
 RaliMetaData
-RALI_API_CALL raliCreateCOCOReader(RaliContext p_context, const char* source_path, bool is_output){
+RALI_API_CALL raliCreateCOCOReader(RaliContext p_context, const char* source_path, bool is_output, bool keypoint, float sigma , int pose_output_width , int pose_output_height){
     if (!p_context)
         THROW("Invalid rali context passed to raliCreateCOCOReader")
     auto context = static_cast<Context*>(p_context);
 
-    return context->master_graph->create_coco_meta_data_reader(source_path, is_output);
+    return context->master_graph->create_coco_meta_data_reader(source_path, is_output, MetaDataReaderType::COCO_META_DATA_READER,  MetaDataType::BoundingBox, keypoint, sigma , pose_output_width , pose_output_height);
+}
```

Comment:
Change this call too. Since its default params you need not send anything.

Reply:
Done